test_runner/fixtures/neon_fixtures.py

```diff
@@ -786,16 +788,17 @@ def cleanup_remote_storage(self):
                     Bucket=self.remote_storage.bucket_name,
                     Delete=objects_to_delete,
                 )
-                objects_to_delete = dict(Objects=[])
+                objects_to_delete = {"Objects": []}
                 cnt += 1
 
         # flush rest
         if len(objects_to_delete["Objects"]):
             self.remote_storage_client.delete_objects(
-                Bucket=self.remote_storage.bucket_name, Delete=objects_to_delete
+                Bucket=self.remote_storage.bucket_name,
+                Delete=objects_to_delete,
             )
 
-        log.info("deleted %s objects from remote storage", cnt)
+        log.info(f"deleted {cnt} objects from remote storage")
```

Comment:
It's better to leave it as is: https://github.com/neondatabase/neon/issues/2866

Reply:
Was trying to query @LizardWizzard about this, and I hope we can still use f-strings in tests, as we don't have any middleware looking at the log statements.

Reply:
Summary:
I would like to keep it the same everywhere. Keeping it as is is one possible way.

I'm not strongly opposed to fstring usage. Would be good to confirm that this solution is preferable for others as well given the limitations.

Reply:
```bash
git grep -E 'log\.(debug|info|error|warn)\("' -- test_runner/  | grep '%s' | wc -l
      11
```

```bash
git grep -E 'log\.(debug|info|error|warn)\(f"' -- test_runner/ | wc -l
     140
```

With this rough estimation, we're closer to using f-strings everywhere for logging